namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-fala-staging/resources/ecr.tf

```diff
@@ -6,16 +6,36 @@ provider "aws" {
   region = "eu-west-1"
 }
 
-module "ecr-repo" {
+module "ecr-repo-laa-fala-webapp" {
   source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
 
   team_name = "laa-get-access"
-  repo_name = "laa-fala"
+  repo_name = "laa-fala-webapp"
 }
 
-resource "kubernetes_secret" "ecr-repo" {
+resource "kubernetes_secret" "ecr-repo-laa-fala-web" {
   metadata {
-    name      = "ecr-repo-laa-fala"
+    name      = "ecr-repo-laa-fala-webapp"
+    namespace = "laa-fala-staging"
+  }
+
+  data {
+    repo_url          = "${module.ecr-repo.repo_url}"
```

Comment:
🙋‍♂️ Given the CI validation error `Error: resource 'kubernetes_secret.ecr-repo-laa-fala-web' config: unknown module referenced: ecr-repo`, we probably need to rename `ecr-repo` to `ecr-repo-laa-fala-web` here as well.